src/main/java/me/botsko/prism/database/PrismDatabaseFactory.java

```diff
@@ -16,10 +15,17 @@
 
     private static PrismDataSource database = null;
 
-    public static void createDefaultConfig(Configuration configuration) {
+    /**
+     * Create a config.
+     * @param configuration ConfigurationSection
+     */
+    public static void createDefaultConfig(ConfigurationSection configuration) {
         ConfigurationSection mysql;
         if (configuration.contains("prism.mysql")) {
             mysql = configuration.getConfigurationSection("prism.mysql");
+            if(mysql == null){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [WhitespaceAround: '{' is not preceded with whitespace.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)